Project.toml

```diff
@@ -13,6 +13,7 @@ LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 LinearMaps = "7a12625a-238d-50fd-b39a-03d52299707e"
 LoopVectorization = "bdcacae8-1622-11e9-2a5c-532679323890"
 MPI = "da04e1cc-30fd-572f-bb4f-1f8673147195"
+Octavian = "6fd5a793-0b7e-452c-907f-f8bfe9c57db4"
```

Comment:
Should get a `[compat]` bound

Reply:
Yeah, sorry - was waiting to see if the CI tests passed (they did not). Will add compat bound soon